docs/install.md

```diff
@@ -349,6 +349,10 @@ data:
   disable-working-directory-overwrite: "true" # Tekton will not override the working directory for individual Steps.
 ```
 
+### Customizing High Availability for the Pipelines Controller
+
+To customize the behavior of HA for the Tekton Pipelines controller, please refer to the related [documentation](/developers/enabling-ha.md).
+
```

Comment:
the link is broken here, has extra `/`, link must point to `developers/enabling-ha.md`

Reply:
fixed